CHANGELOG.md

```diff
@@ -33,32 +33,25 @@ blog:
 
 # 1.18.1
 
-release-date:
-docs:
-blog:
-
-## knownIssues
-
--
+release-date: March 26, 2019
+docs: https://sap.github.io/cloud-sdk/
+blog: https://blogs.sap.com/?p=1068438
 
 ## compatibilityNotes
 
-- Parsing destinations now also checks for validity of the destination. All destinations of type 'HTTP' are expected to contain a `url` or `URL` property. When retrieving a destination from the `destinations` environment variable by name, all destinations are parsed, therefore this validation applies to all available destination, even those that are not read. Invalid destinations will be reported with `Property 'URL' of destination configuration must not be undefined.`
-
-## newFunctionality
-
--
+- Due to the intoroduced validation for destination configurations, destinations of type `HTTP` are expected to contain a `url` or `URL` property. Note that when retrieving a destination from the `destinations` environment variable, all destinations are validated. This validation applies even to destinations that are not read.
 
 ## improvements
 
-- Validate destinations retrieved from environment variable or destination service.
-- Allow setting the format for the keys of a destination in the `destinations` environment variable as known from the destination service - e. g. `URL` in addition to previously only `url`.
+- Introduce validation for destinations retrieved from environment variable or destination service.
+- Allow configuring the `destinations` environment variable with the same property keys as known from the destination service on SAP Cloud Platform, e.g. now `URL` is considered valid in addition to previously only `url`.
 
 ## fixedIssues
 
-- Fix ordering for complex properties including nested complex properties.
+- Fix ordering for properties, including nested complex properties.
```

Comment:
This changes the meaning. I think what was meant is `Fix ordering for properties of type "complex type", that include properties of type "complex type"`.  Not 100% sure though - @FrankEssenberger ?